lightning/src/chain/keysinterface.rs

```diff
@@ -413,7 +425,17 @@ impl InMemoryChannelKeys {
 		}
 	}
 
-	fn remote_pubkeys<'a>(&'a self) -> &'a ChannelPublicKeys { self.remote_channel_pubkeys.as_ref().unwrap() }
+	/// Remote pubkeys
+	/// Will panic if on_accept wasn't called
+	pub fn remote_pubkeys(&self) -> &ChannelPublicKeys { &self.accepted_channel_data.as_ref().unwrap().remote_channel_pubkeys }
+
+	/// Remote to-self delay, constraining local commitment txs
```

Comment:
May be useful to copy the more-detailed comment from remote_to_self_delay from above here too.

Reply:
done